examples_utils/benchmarks/run_benchmarks.py

```diff
@@ -271,15 +269,16 @@ def run_benchmark_variant(
     #     output += analyse_profile(variant_name, cwd)
 
     # Teardown temporary filesystem on all hosts
-    if is_distributed and args.remove_dirs_after:
-        remove_distributed_filesystems(args, poprun_hostnames)
+    if not args.submit_on_slurm:
```

Comment:
These two conditions can go on one line

Reply:
I mean the `args.remove_dirs_after` since its rqeuired in both sub-conditions under this one.

Reply:
And I would have an if + else rather than if + if not, I think some linters might complain too? But this is more a matter of opinion IMO